front/components/assistant/conversation/actions/VisualizationActionIframe.tsx

```diff
@@ -202,7 +201,9 @@ export function VisualizationActionIframe({
       <div
         className={classNames(
           "transition-height relative w-full overflow-hidden duration-500 ease-in-out",
-          codeFullyGenerated ? "min-h-96" : ""
+          codeFullyGenerated && !errored ? "min-h-96" : "",
```

Comment:
🤔 Don't we want the min height only when we have an error?

Reply:
(explained on slack)